src/ucs/Makefile.am

```diff
@@ -5,7 +5,7 @@
 # See file LICENSE for terms.
 #
 
-SUBDIRS = vfs/sock . vfs/fuse
+SUBDIRS = vfs/sock . vfs/fuse nvtx
```

Comment:
maybe put in under src/ucs/profile

Reply:
I was thinking along those lines as well but added this for a draft. I'll change this.